XenModel/XenAPI-Extensions/Session.cs

```diff
@@ -144,9 +144,12 @@ private void LogJsonRequest(string json)
             }
             catch
             {
-                methodName = json;
-                parameters = "";
+                //ignore
             }
+#else
+            methodName = json;
+            parameters = "";

```

Comment:
As `parameters` is always empty in release mode, it would be nice to reflect this on line 162 below so we don't log "with params:" in release mode